src/zebra_puzzles/file_utils.py

```diff
@@ -218,9 +224,9 @@ def prepare_eval_folders(
     # Get sorted names of all prompt files in the data folder
     puzzle_paths = sorted(
         list(
-            Path(f"data/{puzzle_subfolder}/{n_red_herring_clues}rh/puzzles").glob(
-                "*.txt"
-            )
+            Path(
+                f"{data_folder}/{puzzle_subfolder}/{n_red_herring_clues}rh/puzzles"
+            ).glob("*.txt")
```

Comment:
This won't work on Windows - instead, join up paths with "/" rather than including "/" in the strings:

```suggestion
            (Path(data_folder) / puzzle_subfolder / f"{n_red_herring_clues}rh" / "puzzles").glob("*.txt")
```